pkg/cmd/plugincheck2/main.go

```diff
@@ -76,7 +104,17 @@ func main() {
 		defer sourceCodeDirCleanup()
 	}
 
-	diags, err := runner.Check(passes.Analyzers, archiveDir, sourceCodeDir, cfg)
+	diags, err := runner.Check(
+		passes.Analyzers,
+		&analysis.CheckParams{
```

Comment:
This is the change from positional parameters to an object with global data we can pass to the analyzers.